File: static/js/filterMode.js
//개인 필터 버튼을 클릭하면 버튼의 내용을 바꾸고 서버로 모드 정보 전송
function filterMode(self) {
    var mode = ChangeMode(self);
    submitByPost("/filter_mode", 'mode', mode);
}

/**
 * @return {string}
 */
function ChangeMode(self) {
    if (self.value === 'ICO Service 켜기') {
        self.value = 'ICO Service 끄기';
        return 'on';
    }
    else{
        self.value = 'ICO Service 켜기';
        return 'off';
    }
}

File: router/filtering/filter_mode.py
from flask import request, redirect, url_for, session, Blueprint

filter_mode_bp = Blueprint('filter_mode', __name__)


@filter_mode_bp.route('/filter_mode', methods=['POST'])
def filter_mode():
    session['mode'] = request.form['mode']
    return redirect(url_for('news'))


def mode_info():
    if session['mode'] == 'off':
        return 'ICO Service 켜기'
    else:
        return 'ICO Service 끄기'
